src/services/api/search/search.entity.types.ts

```diff
@@ -1,7 +1,10 @@
 export enum SearchEntityTypes {
   USER = 'user',
+  GROUP = 'group', // may be removed
```

Comment:
Is this needed?

Reply:
Yes. There were a couple of duplicate types, so I've created one complete type.
The group is still needed in search v1.